packages/curvenote/src/submissions/submit.utils.ts

```diff
@@ -236,6 +238,41 @@ export async function confirmUpdateToExistingSubmission(
   return kind;
 }
 
+export async function createNewSubmission(
+  session: ISession,
+  logCollector: Record<string, any>,
+  venue: string,
+  kind: string,
+  cdnKey: string,
+  opts?: SubmitOpts,
+) {
+  session.log.debug(`posting new work...`);
+  const { work, workVersion } = await postNewWork(session, cdnKey, session.PRIVATE_CDN);
+  session.log.debug(`work posted with id ${work.id}`);
+
+  session.log.debug(`posting new submission...`);
+  const { submission, submissionVersion } = await postNewSubmission(
+    session,
+    venue,
+    kind,
+    workVersion.id,
+    opts?.draft ?? false,
+  );
+
+  session.log.debug(`new submission posted with id ${submission.id}`);
+
+  if (opts?.draft) {
+    session.log.info(`🚀 ${chalk.green(`Your draft was successfully submitted to "${venue}"`)}.`);
+  } else {
+    session.log.info(`🚀 ${chalk.green(`Your work was successfully submitted to "${venue}"`)}.`);
+  }
+
+  logCollector.work = work;
+  logCollector.workVersion = workVersion;
+  logCollector.submission = submission;
+  logCollector.submissionVersion = submissionVersion;
```

Comment:
now logging the exact same info here that goes into the `transfer.yml`, that means `work.id` etc... rather than `workId`
@rowanc1 not sure if that means you need to make a tweak in the action?

Reply:
No, haven't started using this yet!